app/controllers/api/storage.php

```diff
@@ -386,8 +387,8 @@
         // Map aggregate permissions to into the set of individual permissions they represent.
         $permissions = Permission::aggregate($permissions, $allowedPermissions);
 
-        // Add permissions for current the user if none were provided.
-        if (\is_null($permissions)) {
+        // Add permissions for current the user if none were provided, only if the request is not an admin request or not from console.
+        if (\is_null($permissions) && APP_MODE_ADMIN !== $mode && 'console' !== $project->getId()) {
```

Comment:
These two new conditions will ensure that -
- `APP_MODE_ADMIN !== $mode` - When `x-appwrite-mode` is set to `admin`, user is not assigned. This is set to `admin` in the console.
- `'console' !== $project->getId()` - Although I think this condition will not happen for this API, but just as a sanity check to make sure user from `$dbForConsole` is not used in any situation.